app/perfsymboltable.cpp

```diff
@@ -723,7 +723,29 @@ static QByteArray fakeSymbolFromSection(Dwfl_Module *mod, Dwarf_Addr addr)
     return sym;
 }
 
-static PerfAddressCache::SymbolCache cacheSymbols(Dwfl_Module *module, quint64 elfStart)
+enum Direction{
```

Comment:
sorry, looking at this, it would be simpler to remove this and just call either `symbolAddress` or `alignedAddress` directly